DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: MsExperiment
 Title: Infrastructure for Mass Spectrometry Experiments
-Version: 1.1.4
+Version: 1.5.1
```

Comment:
I'm a bit puzzled by the change from 1.1.x to 1.5.x - no idea why Bioconductor bumped to that version - I thought it should be 1.3.x for devel and 1.2.x for release now...